workspaces/xtermjs/src/package.json

```diff
@@ -10,12 +10,12 @@
   "license": "ISC",
   "dependencies": {
     "command-line-args": "^5.2.1",
-    "command-line-usage": "^6.1.3",
+    "command-line-usage": "^7.0.1",
     "express": "^4.18.2",
     "express-ws": "^5.0.2",
-    "node-gyp": "^9.3.1",
-    "node-pty": "^0.10.1",
-    "xterm": "^5.1.0",
+    "node-gyp": "^9.4.0",
```

Comment:
Is `node-gyp` still required? I know it was previously used to build `node-pty`, but I'm not sure if any other deps depended on it.

Reply:
That's a good question, I'll test it later.

Reply:
Turns out it's not needed. Removed in 9a703c662.